Formula/python@3.7.rb

```diff
@@ -21,6 +21,9 @@ class PythonAT37 < Formula
   # build packages later. Xcode-only systems need different flags.
   pour_bottle? only_if: :clt_installed
 
+  # https://devguide.python.org/versions/#versions
+  deprecate! date: "2023-06-27", because: :deprecated_upstream
```

Comment:
It's already deprecated and in security mode
```suggestion
  disable! date: "2023-06-27", because: :deprecated_upstream
```